src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -14682,6 +14682,21 @@ def test_aks_create_with_acns_performance(
             ],
         )
 
+        # Update transit encryption type to WireGuard and advanced network policies to L7
```

Comment:
Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=138664883&view=results) to validate the change

Reply:
Re-queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=138677564&view=results) with proper subscription.